tests/extraction_tests/test_round_three_extraction.py

```diff
@@ -393,6 +393,8 @@ def test_extract_outcomes(mock_outcomes_sheet, mock_project_lookup, mock_program
     assert project_xor_programme.all()
 
 
+# TODO: Remove skip once when InvalidOutcomeProjectFailure.to_message() is implemented.
```

Comment:
@Joel-Pearce please remove this skip when you complete https://dluhcdigital.atlassian.net/jira/software/c/projects/SMD/boards/140?selectedIssue=SMD-103